docs/version_release_fix.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) 2018 Sony Pictures Imageworks Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import re, os
+
+def main():
+	cur_dir = os.path.dirname(__file__)
```

Comment:
nit: Please use two-space indents instead of tabs, this matches the style used in the rest of the code.

Reply:
fixed whitespace. please check.